### PR TITLE
Soe 745

### PR DESCRIPTION
**Pull request by cjwest**
@boznik : Updated news. please review
I touched 3 modules here:
In this request I pulled the manage content from the views module into its own module (administration)
I updated manage content so the modify operation works 
I updated the content type (stanford_news) so it is no longer overridden when you enable it. (pet peeve)

### CODE REVIEW

**Comment by sherakama** [comment]
This will need an upgrade path since the view moved to a new module. Please provide a hook that enables the new module.

**Comment by cjwest** [comment]
@sherakama  or @boznik,
I added an install hook. Please review.

File: stanford_news.install [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+/**
+ * @file
+ */
+
+
+/**
+ * Enable new Stanford News Administration module.
+ */
+function stanford_news_update_7300($sandbox) {
+    if (!module_exists("stanford_news_administration")) {
```
Comment (sherakama) [inline stanford_news.install]: @boznik, Do you think it is important that we check for a modified view first? Ie: the view is in the DB? I actually think NOT checking for it is the way we want to go. That should keep it consistent with how its relationship with features is currently. Thoughts?

Comment (boznik) [inline stanford_news.install]: yes, let's not check.

Comment (sherakama) [inline stanford_news.install]: Cool, It's your PR to merge when you are ready.

**Comment by cjwest** [comment]
W00t!

**Pull request merged**